Neos.Flow/Classes/Http/RequestHandler.php

```diff
@@ -92,15 +91,17 @@ public function handleRequest()
     {
         // Create the request very early so the ResourceManagement has a chance to grab it:
         $request = ServerRequest::fromGlobals();
+        // TODO: Required for b/c - can be removed with Flow 7
         $response = new \GuzzleHttp\Psr7\Response();
         $this->componentContext = new ComponentContext($request, $response);
+        $this->bootstrap->setEarlyInstance(ComponentContext::class, $this->componentContext);
```

Comment:
This makes the Context injectable and is needed to make the `MiddlewaresChainFactory` be instanciable via object management.